src/src/AdminBundle/Resources/views/Event/base.html.twig

```diff
@@ -0,0 +1,2 @@
+{% extends "@AdminBundle/Resources/views/base_admin.html.twig" %}
```

Comment:
potrebuje `Event` tento subor? nestaci ked to bude dedit z `EventType`?

Reply:
edit.html.twig moze dedit z EventType. To dava zmysel. Ale edit musi dedit z base-u. A index.html twig je nepotrebny.

Reply:
Moja otazka skor bola, ci `edit.html.twig` a `create.html.twig` nemozu dedit z `src/src/AdminBundle/Resources/views/EventType/base.html.twig` kedze ten je rovnaky ako tento subor.

Reply:
Hmm, tak potom naco je base.html.twig v kazdom priecinku, ked moze byt jeden vseobecny? 🤔

Reply:
pozri si ake su tam rozdiely v tych suboroch a isto na to prides :)

Reply:
kvoli `active`

Reply:
Okej, bude to dedit z toho co si napisal